### PR TITLE
Added german translation

### PR DESCRIPTION
**Pull request by senk**
Still some errors and untranslated variables. 
Are you guys willing to add the german language?

Edit: finished translating

### CODE REVIEW

**Comment by bobbyjunzhang** [comment]
We would love to have the German language! Thanks for the great efforts. We are reviewing it now. Kindly let us know what are the 'errors and untranslated variables' - we want to fix them also.

**Comment by senk** [comment]
Currently the the project description that appears on the front page is untranslated. The issue I'm facing currently is that I cannot click the login button after switching to German could be something related to the German umlaute. I will look into this later today

File: static/resources/js/common.js [hunk 1]
```diff
@@ -69,6 +69,7 @@ AjaxUtil.prototype.exec = function(){
 var SUPPORT_LANGUAGES = {
 	"en-US": "English",
 	"zh-CN": "Chinese"
+	"de-DE": "German"
```
Comment (wknet123) [inline static/resources/js/common.js]: Missing a comma

File: static/resources/js/common.js [hunk 2]
```diff
@@ -69,6 +69,7 @@ AjaxUtil.prototype.exec = function(){
 var SUPPORT_LANGUAGES = {
 	"en-US": "English",
 	"zh-CN": "Chinese"
```
Comment (wknet123) [inline static/resources/js/common.js]: Missing a comma

Comment (senk) [inline static/resources/js/common.js]: fixed it, thanks!

**Comment by senk** [comment]
finished translating, ready for merge

**Comment by wknet123** [comment]
Hi, @senk since I don't know about German, I appreciate you to confirm your translations are properly before we merge it.

**Comment by senk** [comment]
I think they're ready to be merged. Suggestions for improvement are welcomed in the future 😉

**Comment by wknet123** [comment]
OK, thanks again.

**Comment by senk** [comment]
@AlexZeitler maybe you could take a look at the translation and see if it works for you 😃

**Comment by AlexZeitler** [comment]
@senk Sorry for the late response...

In general it looks fine but there is a mix of "Sie" and "Du". If there is no particular reason for that it might be fixed.

**Pull request merged**